packages/amplication-git-pull-request-service/src/core/diff/diffset-mapper.ts

```diff
@@ -0,0 +1,26 @@
+import { DiffSet } from 'dir-compare';
+import { PrModule } from '../../constants';
+import { DiffVisitorFn } from './visitors/types';
+
+export function mapDiffSetToPrModule(
```

Comment:
```suggestion
/**
 * This method takes the diffSets and visitor functions and returns an array of modules.
 * We use the visitor pattern here (https://en.wikipedia.org/wiki/Visitor_pattern), which means
 * this function can be provided with any amount of "visitors". Each visitor function will be executed
 * and provided with each diffset.
 * 
 * Empty results (null/undefined) will not be included in the final result.
 */
export function mapDiffSetToPrModule(
```

I believe this function deserves an explanation.